src/main/flight/mixer_init.c

```diff
@@ -268,7 +268,7 @@ const mixer_t mixers[] = {
     { 1, true,  NULL },                // MIXER_SINGLECOPTER
     { 4, false, mixerAtail4 },         // MIXER_ATAIL4
     { 0, false, NULL },                // MIXER_CUSTOM
-    { 2, true,  NULL },                // MIXER_CUSTOM_AIRPLANE
+    { 1, true,  NULL },                // MIXER_CUSTOM_AIRPLANE
```

Comment:
IMO this will prevent 2 motors on custom airplane.

Reply:
hmm somehow it supposed to work with zero for `{ 0, false, NULL }, // MIXER_CUSTOM`...

Reply:
Already suggested to add MIXER_CUSTOM_WING instead.

Reply:
i don't understand why thats needed. Shouldn't the CUSTOM_AIRPLANE cover all the possible configurations?

Reply:
Just following documentation and original firmware code - which has been refactored years ago - would like to know the reason for 2 motors used. That's why the suggestion for a fully custom mixer mode for wings.

https://betaflight.com/docs/development/mixer

Reply:
yeah i don't see the reason why it should be 2 motors. As far as I understand Wez used it with 4 motors even (CUSTOM_AIRPLANE). I think we can shange it to minimum 1 motor, and then no need to CUSTOM_WING. Cuz that will be confusing, WING theoretically can have two motors as well.

Reply:
@ledvinap is right. Just tested. `CUSTOM_AIRPLANE` now only allows one output. `CUSTOM_WING` works as expected (added model in configurator for testing).

Reply:
Hmm how did you check it? Like it doesn't allow add more motors with Mmix, or it allows, but doesn't spin them? I think it's better to fix this behavior then

Reply:
in this function can see that for `CUSTOM_xxx` it should just ignore the default motor count and pull the motor count and motors from mmix:
https://github.com/betaflight/betaflight/blob/master/src/main/flight/mixer_init.c#L414

Reply:
Switching between these modes it not working as expected. Going to QUAD_X and then a custom one is different

CUSTOM_xxx requires a custom mmix

![image](https://github.com/user-attachments/assets/a36a8462-b30a-47cd-9332-8364d7a444d2)

while 4 motor outputs are available - but entering tab again in this mode motors are zero.

After selecting FLYWING_WING (after resetting to QUAD_X) it first shows 4 motors. Re-enter tab and it shows 1 motor as suspected.